### ISSUE TITLE
Debian 10 crash on launch

### ISSUE THREAD

**Issue by bluedalmatian** [casinocoin/casinocoin-token-wallet#2]
Installed from official .deb package.

launching it from the gui  it just exitswithout displaying anything.

 so i launched it from the cli to get some error output. first of all i got a message saying "[4589:0805/134318.458187:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /opt/CasinoCoin Token Wallet/chrome-sandbox is owned by root and has mode 4755.
Trace/breakpoint trap" i set the permissions to 4755 but now it just exits with "Trace/breakpoint trap"

OS: Debian 10 amd64

**Comment by bluedalmatian** [comment]
Also tested the RPM on CentOS 7, exactly the same issue

**Comment by bluedalmatian** [comment]
Exit status appears to be 133 if that helps

**Comment by t3nshi** [comment]
first let me make sure of some things... you said you installed correctly and it is crashing on launching. Can you confirm that you are using a window manager on X or on Wayland and that you are launching as a user and not as root? It would help a lot since if you give the process with a little bit more accuracy I can reproduce your problem. I use ubuntu without issues, so it should work seamlessly in Debian. In the mean time I will roll out a Debian and a CentOS (ugh!) and I will try to see the issue.

**Comment by bluedalmatian** [comment]
yes this is with a default installation of Gnome 3 on Wayland Debian 10 launched as a non root user.
cent os 7 is also gnome 3 but im not sure if that is using wayland or x

**Comment by ghost** [comment]
I can confirm a similar behavior

- Fresh install of Debian 10.8 with with Gnome with x11
- Install libappindicator3-1 
- Wallet version 5.0.8 as non-root user
- Error received when trying to launch there is no error prompt or GUI. Just the following in the syslog:
````
Mar  5 13:24:22 tycho kernel: [  123.459707] traps: casinocoin-toke[1735] trap int3 ip:5566ec07ca9f sp:7fff2c77b4f0 error:0 in casinocoin-token-wallet[5566e8c8f000+53d2000]
````
Running as root
````
root@tycho:~# casinocoin-token-wallet --no-sandbox 
No protocol specified

(casinocoin-token-wallet:1676): Gtk-WARNING **: 13:41:55.313: cannot open display: :0
````
I did confirm there are no issues on:
- ubuntu 18.04.5
- ubuntu 20.04.1

I believe this is an issue for electron support on wayland as its now the default on Buster. I will switch over to Ubuntu as its has LTS support (as much as I love Debian).
